apps/examples/testcase/ta_tc/mqtt/utc/utc_mqtt_main.c

```diff
@@ -0,0 +1,483 @@
+	TC_SUCCESS_RESULT();
+}
+
+
+/**
+ * @testcase         utc_mqtt_unsubscribe_n
+ * @brief            stop subscribing from a broker
+ * @scenario         call mqtt_unsubscribe with an invalid input
+ * @apicovered       mqtt_unsubscribe
+ * @precondition     mqtt_subscribe
+ * @postcondition    none
+ */
+static void utc_mqtt_unsubscribe_n(void)
+{
+	int res = mqtt_unsubscribe(NULL, NULL);
+	TC_ASSERT_LT("mqtt_unsubscribe", res, 0);
+	TC_SUCCESS_RESULT();
+}
+
+
+/**
+ * @testcase         utc_mqtt_unsubscribe_p
+ * @brief            stop subscribing from a broker
+ * @scenario         call mqtt_unsubscribe
+ * @apicovered       mqtt_unsubscribe
+ * @precondition     mqtt_subscribe
+ * @postcondition    none
+ */
+static void utc_mqtt_unsubscribe_p(void)
+{
+	int res = mqtt_unsubscribe(g_mqtt_sub_handle, UTC_MQTT_TOPIC);
+	TC_ASSERT_EQ("mqtt_unsbscribe", res, 0);
+	TC_SUCCESS_RESULT();
+}
+
+
+/**
+ * @testcase         utc_mqtt_disconnect_n
+ * @brief            disconnect to broker
+ * @scenario         disconnect to broker using an invalid handle
+ * @apicovered       mqtt_disconnect
+ * @precondition     mqtt_connect
+ * @postcondition    none
+ */
+static void utc_mqtt_disconnect_n(void)
+{
+	int res = mqtt_disconnect(NULL);
+	TC_ASSERT_LT("mqtt_disconnect", res, 0);
+	TC_SUCCESS_RESULT();
+}
+
+
+/**
+ * @testcase         utc_mqtt_disconnect_p
+ * @brief            disconnect to broker
+ * @scenario         disconnect to broker using a valid handle
+ * @apicovered       mqtt_disconnect
+ * @precondition     mqtt_connect
+ * @postcondition    none
+ */
+static void utc_mqtt_disconnect_p(void)
+{
+	int res = mqtt_disconnect(g_mqtt_sub_handle);
+	TC_ASSERT_EQ("mqtt_disconnect", res, 0);
+
+	UTC_MQTT_WAIT_SIGNAL;
+	TC_SUCCESS_RESULT();
+}
+
+
+/**
+ * @testcase         utc_mqtt_deinit_client_n
+ * @brief            free all resources used to mqtt stack
+ * @scenario         call mqtt_deinit with an invalid input
+ * @apicovered       mqtt_deinit_client
+ * @precondition     mqtt_init
+ * @postcondition    none
+ */
+static void utc_mqtt_deinit_client_n(void)
+{
+	int res = mqtt_deinit_client(NULL);
+	TC_ASSERT_LT("mqtt_deinit_client", res, 0);
+	TC_SUCCESS_RESULT();
+}
+
+
+/**
+ * @testcase         utc_mqtt_deinit_client_p
+ * @brief            free all resources used to mqtt stack
+ * @scenario         call mqtt_deinit with an invalid input
+ * @apicovered       mqtt_deinit_client
+ * @precondition     mqtt_init
+ * @postcondition    none
+ */
+static void utc_mqtt_deinit_client_p(void)
+{p
+	int res = mqtt_deinit_client(g_mqtt_pub_handle);
+	if (res < 0) {
+		UTC_MQTT_LOGE;
+	}
+	res = mqtt_deinit_client(g_mqtt_sub_handle);
+	TC_ASSERT_EQ("mqtt_deinit_client", res, 0);
+	TC_SUCCESS_RESULT();
+}
+
+
+static int mqtt_utc(int argc, FAR char *argv[])
+{
+	sem_wait(&tc_sem);
+	working_tc++;
+
+	total_fail = total_pass = 0;
+
+	printf("=== TIZENRT MQTT TC START! ===\n");
+	int res = _utc_mqtt_init();
+	if (res < 0) {
+		UTC_MQTT_LOGE;
+		return -1;
```

Comment:
After starting of tc, we should print a result of tc. Let's change it like belows.
```
	printf("=== TIZENRT MQTT TC START! ===\n");
	int res = _utc_mqtt_init();
	if (res < 0) {
		UTC_MQTT_LOGE;
		total_fail = 1;
		goto exit;
	}
	....
exit:
	printf("\n=== TIZENRT MQTT TC COMPLETE ===\n");
....
```

Reply:
done